index.js

```diff
@@ -6,52 +6,20 @@ import './shim.js';
 
 import React from 'react';
 import { AppRegistry, StatusBar } from 'react-native';
-import SplashScreen from 'react-native-splash-screen';
 
 import App from './App';
-import WalletMigrate from './walletMigrate';
 
 if (!Error.captureStackTrace) {
   // captureStackTrace is only available when debugging
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
   Error.captureStackTrace = () => {};
 }
 
-class BlueAppComponent extends React.Component {
-  state = {
-    isMigratingData: true,
-  };
-
-  componentDidMount() {
-    const walletMigrate = new WalletMigrate(this.setIsMigratingData);
-
-    walletMigrate.start();
-  }
-
-  setIsMigratingData = async () => {
-    SplashScreen.hide();
-    this.setState({
```

Comment:
how do you hide the splashscreen then?

Reply:
the application seems to work fine without it

Reply:
clean build and you should see eternal splashScreen. SplashScreen.hide() needs to be called in the root component.
https://github.com/crazycodeboy/react-native-splash-screen#usage

Reply:
added